run/markdown-preview/editor/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "markdown-preview",
+  "description": "Cloud Run service to demonstrate service-to-service authentication, paired with Renderer service.",
+  "version": "0.0.1",
+  "private": true,
+  "license": "Apache-2.0",
+  "author": "Google LLC",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/GoogleCloudPlatform/nodejs-docs-samples.git"
+  },
+  "engines": {
+    "node": ">= 12.0.0"
+  },
+  "main": "main.js",
+  "scripts": {
+    "e2e-test": "test/runner.sh mocha test/system.test.js --timeout=30000",
+    "start": "node main.js",
+    "test": "mocha test/*.test.js --exit"
```

Comment:
npm test will also try to run the `test/system.test.js`

Reply:
Fixed